CHANGELOG.md

```diff
@@ -38,6 +38,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - String based `ocalls`/`ecalls`, `OE_ECALL`, and `OE_OCALL` macros.
 
+[v0.4.1] - 2018-12-21
+---------------------
+
+v0.4.1 contains a small fix to work with Intel's new ISV version bump.
+
+### Changed
+
+- Changed the ISV version check to be greater than or equal to 1, instead of exactly 1.
+
```

Comment:
To be more specific, this change is to make OE SDK not only allow reports signed by QE SVN=1 but also allows newer QE SVN (greater than 1) during oe_verify_report process.